code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/LdapConfig.java

```diff
@@ -0,0 +1,153 @@
+package io.cattle.platform.iaas.api.auth.integration.ldap;
+
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfig;
+import io.github.ibuildthecloud.gdapi.annotation.Field;
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+@Type(name = LdapConstants.CONFIG)
```

Comment:
The API URLs are normally case-insensitive so I don't think you would break anything if you made the returned ones be {github,ldap}Config.